paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/settings/PlaygroundSettingDefinition.kt

```diff
@@ -54,6 +72,27 @@ internal interface PlaygroundSettingDefinition<T> {
         }
     }
 
+    @OptIn(ExperimentalCustomerSheetApi::class)
+    data class CustomerSheetConfigurationData(
+        private val configurationBuilder: CustomerSheet.Configuration.Builder,
+        var billingDetailsCollectionConfiguration: PaymentSheet.BillingDetailsCollectionConfiguration =
```

Comment:
I think this should be `private val`, not `var`.

Reply:
Talked in Slack, is a `var` as it's needed to update each address definition individually.